sqrl-planner/src/main/java/com/datasqrl/engine/log/postgres/PostgresPhysicalPlan.java

```diff
@@ -0,0 +1,14 @@
+package com.datasqrl.engine.log.postgres;
+
+import com.datasqrl.engine.EnginePhysicalPlan;
+import com.datasqrl.sql.SqlDDLStatement;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Value;
+
+@Value
+public class PostgresPhysicalPlan implements EnginePhysicalPlan {
+  List<SqlDDLStatement> ddl = new ArrayList<>();
+  List<SqlDDLStatement> listenQueries = new ArrayList<>();
+  List<SqlDDLStatement> onNotifyQueries = new ArrayList<>();
```

Comment:
might be better to have a set off ddl and a set of 'queries' that include a listen and onNotify pair. We can keep the association and would be easier to build the physical plan on the server when we move it to calcite.

Reply:
good idea, I changed it